acceptance-tests/src/test/java/uk/gov/di/test/controllers/UserInterventionLifecycleController.java

```diff
@@ -0,0 +1,92 @@
+package uk.gov.di.test.controllers;
+
+import uk.gov.di.test.entity.UserInterventions;
+import uk.gov.di.test.entity.UserProfile;
+import uk.gov.di.test.services.DynamoDbService;
+import uk.gov.di.test.services.TestConfigurationService;
+import uk.gov.di.test.utils.Crypto;
+
+public class UserInterventionLifecycleController {
```

Comment:
Consider naming along the same lines as our api code - maybe this is UserInterventionDynamoService or something similar? Same for other controllers